Dockerfile

```diff
@@ -1,6 +1,17 @@
+FROM golang:latest AS build-image
```

Comment:
I think we should avoid using latest image.

Reply:
Yes,use go version 1.12.